tests/dtm/test_single_process_restart.py

```diff
@@ -171,3 +178,122 @@ def test_write_during_m0d_restart(self):
 
         self.test_completed = True
         self.log.info("ENDED: Verify READ during m0d restart using pkill")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-41232")
+    def test_copy_object_after_m0d_restart(self):
+        """Verify copy object after m0d restart using pkill."""
+        self.log.info("STARTED: Verify copy object after m0d restart using pkill")
+        bucket_name_1 = 'bucket-test-41232-1'
+        bucket_name_2 = 'bucket-test-41232-2'
+        object_name = 'object-test-41232'
+        self.log.info("Step 1: Start write Operations :")
+        resp = self.s3_obj.create_bucket(bucket_name=bucket_name_1)
+        assert_utils.assert_true(resp[0], resp[1])
+        resp = self.s3_obj.create_bucket(bucket_name=bucket_name_2)
+        assert_utils.assert_true(resp[0], resp[1])
```

Comment:
Please format bucket name strings and create buckets in loop.

Reply:
Changed to loop and bucket name is fetched from bucket_list (s3 lib function)